docs/ReleaseNotes.md

```diff
@@ -18,7 +18,7 @@ Major Changes :race_car: :red_car: :blue_car:
 Bug Fixes:bug:
 
 | Feature                                  | Brief summary                                                                                                                                 | Category                                           | Pull request                                                      | Contributor                                   |
-| ---------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------- | ----------------------------------------------------------------- | --------------------------------------------- |
+|------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------|-------------------------------------------------------------------|-----------------------------------------------|
```

Comment:
Please don't create diffs that depend on the formatter settings only in your specific environments.
```suggestion
| ---------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------- | ----------------------------------------------------------------- | --------------------------------------------- |
```

Reply:
ops thank you for the comment!
I will format with default setting at the end, sorry
(I used formatter setting for autoware : just execuse :bow: :bow: :bow:

Reply:
@kyoichi-sugahara You can modify it by using commit suggestion.